docs/schemaform/form-config.md

```diff
@@ -66,9 +66,23 @@ Forms are created by creating a page that uses FormApp from the schemaform folde
           // Objects that start with view: will not be sent, but their children will be merged
           // into the parent object and will be sent
           // These can be used to remove fields from what is sent to the server, like if we have
-          // a question that is only used to reveal other questions
+          // a question that is only used to reveal other questions or group related questions
+          // together to be conditionally revealed that aren't in an object in the schema
```

Comment:
We could add use cases to the cookbook section. Like, "I want to conditionally hide a group of fields" and "I want to group fields together to validate them together" (used for email on the contact info page).